challenges/arrays-callbacks.js

```diff
@@ -30,22 +35,30 @@ The zoos need a list of all their animal's names (animal_name only) converted to
 */
 
 const lowCaseAnimalNames = [];
+zooAnimals.map(function(currentValue){
+  lowCaseAnimalNames.push(
+    currentValue.animal_name.toLowerCase());
+});
 console.log(lowCaseAnimalNames);
 
 /* Request 3: .filter() 
 
 The zoos are concerned about animals with a lower population count. Using filter, create a new array of objects called lowPopulationAnimals which contains only the animals with a population less than 5.
 
 */
-const lowPopulationAnimals = [];
+const lowPopulationAnimals  = zooAnimals.filter(function(currentValue){
```

Comment:
I really like how you removed the step of assigning lowPopulationAnimals to an empty array. It is much more concise this way.